monumenten.map

```diff
@@ -41,7 +41,7 @@ MAP
       "wfs_adstract"        "Monument coordinaten Amsterdam"
       "wfs_enable_request"  "*"
       "gml_featureid"       "ID"
-      "gml_include_items"   "all"
+      "gml_include_items"   "id,external_id,monument_aanwijzingsdatum,in_onderzoek,monumentnummer,monumentnaam,display_naam,betreft_pand,heeft_als_grondslag_beperking,monumentstatus,geometry"
```

Comment:
Handig dat je dat hier al kunt filteren.

Reply:
Dan heb je geen view nodig in postgres neem ik aan?

Reply:
Nee, dat is dan niet nodig . Ik heb het uitgeprobeerd en zo zie je alleen de included items